### ISSUE TITLE
Hypergrid React Component

### ISSUE THREAD

**Issue by Dwaynekj** [fin-hypergrid/core#343]

**Comment by ndrsn** [comment]
What is the status on this? Is there a set of requirements available? I have written a small wrapper component for internal use, but if clear on what direction to go I wouldn't mind working on a more generic component.

**Comment by Dwaynekj** [comment]
@ndrsn Thank for the interest. We have plans to reassess this later near the end of the year. We are moving alot of things in the core of the grid, so it won't be wise to tackle this feature as of yet. I'll remove the on-hold status when we are ready

**Comment by Dwaynekj** [comment]
@ndrsn Just an update that we haven't forgotten about React support. Our current focus has been on updating documentation to encourage more contributors. In the interim we are open to any exploration you can offer

**Comment by aqumus** [comment]
Question: Could we add custom react components for custom cell rendering where grid is already inside a react container

**Comment by Dwaynekj** [comment]
Do you mean a custom cell render react component?

**Comment by aqumus** [comment]
Yes, could we provide custom template for cell where the template wud be a react element.

**Comment by Dwaynekj** [comment]
yes sure!

**Comment by aqumus** [comment]
How could we acheive rendering of react component in a particular cell ?
or define a cell renderer (which renders a React component) for a particular column?

**Comment by Dwaynekj** [comment]
Sorry, I might have spoken to soon the last time. React would be a good use case for a cell editor which are DOM based. For cell editors, we suggest you make a child class from renderers/EmptyCell  . Note that cell renderers use canvas are painted anytime any cell with that renderer is visible and a renderer.paintCells is called. I do not believe React will be effective for making canvas based components, please correct if I'm wrong
